charts/kserve-resources/templates/configmap.yaml

```diff
@@ -359,6 +362,16 @@ data:
 
            # cpuLimit is the limits.cpu to set for the router container.
            "cpuLimit": "1"
+
+           # Propagate the specified headers to all the steps specified in an InferenceGraph.
+           # You can either specify the exact header names or use Golang supported regex patterns to propagate multiple headers.
```

Comment:
Let's add inline link here in this comment to "Golang supported regex patterns" docs/github